src/bowling.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+class Bowling{
+  constructor() {
+    this.score = [];
+  };
+
+  scoring(num) {
+    this.score.push(num);
+  };
+
+  frame1() {
+    if(this.score[0] + this.score[1] === 10) {
+            return this.score[0] + this.score[1] + this.score[2];
+        }
+    return this.score[0] + this.score[1];
+  };
+
+  frame2() {
+    if(this.score[2] + this.score[3] === 10) {
+            return this.score[2] + this.score[3] + this.score[4] + this.frame1();
+        }
+    return this.score[2] + this.score[3] + this.frame1();
+  };
```

Comment:
Consider refactoring to put the separate functions into one function.

Reply:
Maybe send the frame number as an argument to your frame method to avoid repetition.